src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.util.List;
+import model.FruitTransaction;
+import service.FileReaderService;
+import service.FileWriterService;
+import service.ReportService;
+import service.impl.FileReaderServiceImpl;
+import service.impl.ReportServiceImpl;
+import service.impl.ReportWriterService;
+
+public class Main {
+    private static final FileReaderService FILE_READER_SERVICE = new FileReaderServiceImpl();
+    private static final ReportService REPORT_SERVICE = new ReportServiceImpl();
+    private static final FileWriterService FILE_WRITER_SERVICE = new ReportWriterService();
```

Comment:
create and init your services directly in main method

Reply:
Done